### DISCUSSION TITLE
Need Help to setup the project locally

### DISCUSSION THREAD

**Discussion by MAYANK12SHARMA** [JdeRobot/RoboticsAcademy#2662] [Q&A] [answered]
When i try to setup the project locally then i am facing this error
![image](https://github.com/user-attachments/assets/b1991979-fffe-4b06-a4b0-c142b8190095)

Actually when run the command **npm run build**  at the react-frontend directory then a error getting which is 
![image](https://github.com/user-attachments/assets/b45b1cf3-05b6-4619-991f-7c71817ce0cb)
![image](https://github.com/user-attachments/assets/de02bbaf-21ba-4973-91da-5ff83e43f446)

**Comment by javizqh** [comment]
Hello, what guide are you using for setting up your project?

**Reply by SujalRaunak** [reply]
@MAYANK12SHARMA @javizqh I am facing same problem. How to fix it. Need some help.

**Reply by javizqh** [reply, marked as answer]
Make sure the version of nvm is the 16. You can execute these commands found in [here](https://github.com/JdeRobot/RoboticsAcademy/blob/humble-devel/docs/InstructionsForDevelopers.md) :
```bash
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
nvm install 16
nvm use 16
cd react_frontend/ && yarn install && yarn run dev
```
